packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-get-att.yaml

```diff
@@ -0,0 +1,32 @@
+Resources:
+  Bucket1:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: SomeBucketName
+      ReplicationConfiguration:
+        Role: "some-role"
+        Rules:
+        - Destination:
+            Bucket: Bucket1
+            StorageClass: STANDARD
+          Status: Enabled
+  Bucket2:
```

Comment:
How about we have `Bucket1` and `Bucket2`, and we just use different ways of referencing attributes of  `GetAtt` of `Bucket1` in the properties of `Bucket2`? This test's length would probably be cut in half, while retaining the same coverage.